simtools/applications/tune_psf.py

```diff
@@ -276,8 +278,11 @@ def run_pars(pars, plot=True):
         im = ray.images()[0]
         d80 = im.get_psf()
 
-        # Simulated cumulative PSF
-        data_to_plot["simulated"] = im.get_cumulative_data(radius * u.cm)
+        if radius is not None:
```

Comment:
I think we violate here our aim of 'locality': the error should be thrown in get_cumulative_data.

Reply:
Yes right, I removed it. It's already inside `get_cumulative_data`.